edit-curriculum.md

```diff
@@ -24,7 +24,12 @@ The Add and Edit Curriculum screens provide interfaces for editing an existing c
 	* Academic area for which the curriculum is set up (each curriculum can only have one academic area)
 
 * **Major(s)**
-	* The major (or more majors) within the selected academic area for which the curriculum is set up 
+	* In Add Curriculum:
+		* Allows for defining curricula with multiple majors
+			* In this case, curricula from each major are inhereted and can be overridden on a multiple major level (when multiple majors are selected)
+			* Common courses can be similarly defined on a curriculum with no major
+	* In Edit Curriculum:
+		* The major (or more majors) within the selected academic area for which the curriculum is or will be set up 
 
```

Comment:
This is not correct! The Add Curriculum comment belongs to the Multiple Majors toggle, which exists on both add and edit pages.